internal/pkg/cli/svc_delete.go

```diff
@@ -80,11 +89,20 @@ func newDeleteSvcOpts(vars deleteSvcVars) (*deleteSvcOpts, error) {
 		sess:    sessProvider,
 		sel:     selector.NewConfigSelector(prompter, store),
 		appCFN:  cloudformation.New(defaultSession, cloudformation.WithProgressTracker(os.Stderr)),
-		getSvcCFN: func(session *awssession.Session) wlDeleter {
-			return cloudformation.New(session, cloudformation.WithProgressTracker(os.Stderr))
+		getSvcCFN: func(sess *awssession.Session) wlDeleter {
+			return cloudformation.New(sess, cloudformation.WithProgressTracker(os.Stderr))
 		},
-		getECR: func(session *awssession.Session) imageRemover {
-			return ecr.New(session)
+		getECR: func(sess *awssession.Session) imageRemover {
+			return ecr.New(sess)
+		},
```

Comment:
lol why is this even in the diff

Reply:
that's a great point....i have no idea :joy: